src/views/Restoration/RestorationBusinessName.vue

```diff
@@ -39,12 +39,16 @@
       </header>
       <v-card
         flat
-        class="mt-5"
+        class="mt-4"
       >
         <RestorationType id="restoration-type" />
+
+        <!-- Divider b/w Restoration and Approval type -->
+        <v-divider/>
+
         <ApprovalType
           id="approval-type"
-          class="mt-n10"
+          class="mt-4"
```

Comment:
I understand the need to have specified top whitespace, but you've removed bottom negative whitespace (which was used to make this component "shorter"). Is that no longer needed?

Reply:
Per my screenshot above, it doesn't look like ApprovalType needs to be any shorter.